src/modules/mavlink/mavlink_messages.cpp

```diff
@@ -3958,7 +3958,7 @@ class MavlinkStreamManualControl : public MavlinkStream
 			msg.target = mavlink_system.sysid;
 			msg.x = manual_control_setpoint.x * 1000;
 			msg.y = manual_control_setpoint.y * 1000;
-			msg.z = manual_control_setpoint.z * 1000;
+			msg.z = (manual_control_setpoint.z + 1.f) * .5f * 1000;
```

Comment:
Shouldn't this stay as is?

Reply:
I agree, I only changed it since I'm not sure if something breaks.